api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -83,16 +83,13 @@ model ClusterSpec {
   api: ApiProfile;
 
   /** Enable FIPS mode for the cluster
-   * When set to true, `etcdEncryption` must be set to true
+   * When set to true, `etcdProfile.encryptionProfile.kms` must be set
```

Comment:
This is relevant. The reason I documented this is the following:

In CS, if the "fips" attribute is set then *etcd encryption with kms-based user provided encryption keys* is mandatory.

Let me know if there's something to be taken into consideration here.

Reply:
Do we know why that's a requirement?